databuilder/transformer/table_tag_transformer.py

```diff
@@ -0,0 +1,31 @@
+from pyhocon import ConfigFactory
+
+from databuilder.transformer.base_transformer import Transformer
+from databuilder.models.table_metadata import TableMetadata
+
+
+class TableTagTransformer(Transformer):
```

Comment:
would be good to put a doc on https://github.com/lyft/amundsendatabuilder#list-of-transformers as well as some  unit tests.